web/impact/impact/routers.py

```diff
@@ -18,3 +24,31 @@ def allow_migrate(self, db, app_label, model_name=None, **hints):
                 'contenttypes']:
             return False
         return True
+
+
+class MasterSlaveAPIRouter(MasterSlaveRouter):
+
+    def allow_migrate(
+            self,
+            db,
+            app_label,
+            model_name=None,
+            **hints):
+        """
+        Make sure that we don't migrate
+        apps shared by both accelerate and impact
+        """
+        allow_migrate = super().allow_migrate(
+            db, app_label,
+            model_name,
+            **hints)
+
+        if app_label in [
+                'auth',
+                'admin',
+                'sessions',
+                'simpleuser',
+                'sites',
+                'contenttypes']:
```

Comment:
I'd prefer to see this if statement dried out. But since we're removing it soon (@jonkiparsky correct me if I'm wrong about this assumption), not blocking.

Reply:
Just for protocol - 
This list appears in: https://github.com/masschallenge/impact-api/pull/99/files#diff-4fbc2fef212daea86092e335e9888262R18

and in: https://github.com/masschallenge/impact-api/pull/99/files#diff-4fbc2fef212daea86092e335e9888262R46